estate/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    'name': "estate app",
+    'summary': "estate module",
+    'description': "estate module for buying and selling properties. Users can buy/sell properties via estate agents",
+    'author': "Odoo",
+    'category': 'Tutorials/estate',
+    'version': '0.1',
```

Comment:
```suggestion
    'version': '1.0',
```
In version `x.y`, x refers to main version and y is for update we generally start versioning of module from 1.0